.gitlab/download-single-step-artifacts.sh

```diff
@@ -10,7 +10,7 @@ if [ -n "$CI_COMMIT_TAG" ] || [ -n "$DOTNET_PACKAGE_VERSION" ]; then
   echo "Downloading artifacts from Github"
   VERSION=${DOTNET_PACKAGE_VERSION:-${CI_COMMIT_TAG##v}} # Use DOTNET_PACKAGE_VERSION if it exists, otherwise use CI_COMMIT_TAG without the v
 
-  for SUFFIX in "" ".arm64"; do
+  for SUFFIX in "" ".arm64" ".musl"; do
```

Comment:
I don't think this is correct 🤔 

The packages are called 
- `datadog-dotnet-apm-2.56.0-musl.tar.gz`
- `datadog-dotnet-apm-2.56.0.tar.gz`
- `datadog-dotnet-apm-2.56.0.arm64.tar.gz`

So I think you need `"-musl"` as the _source_ suffix

I suspect you'll also need to update something else to account for this, `prepare-oci-package.sh` maybe?

Reply:
fixed